dashboard/test/factories/pd_factories.rb

```diff
@@ -115,15 +115,15 @@
   end
 
   factory :regional_partner_newjersey, parent: :regional_partner_with_summer_workshops do
-    # No contact details, and no workshop application dates.
+    # No contact details, no workshop application dates, and no workshops.
     contact_name nil
     contact_email nil
     apps_open_date_csp_teacher nil
     apps_open_date_csd_teacher nil
     apps_close_date_csp_teacher nil
     apps_close_date_csd_teacher nil
     mappings {[create(:pd_regional_partner_mapping, state: "NJ")]}
-    pd_workshops {[create(:pd_workshop, :local_summer_workshop_upcoming)]}
+    pd_workshops {[]}
```

Comment:
If we're not overriding the default, this shouldn't be needed at all. Remove this line.

Reply:
Its parent actually [does](https://github.com/code-dot-org/code-dot-org/blob/fee01717bab098398c90e88fa7456942a095251e/dashboard/test/factories/factories.rb#L1162) create workshops, as this one factory is the exception, not the norm.